internal/cmd/launcher.go

```diff
@@ -335,24 +335,27 @@ func isAnalysisRequired(cmd *cobra.Command) bool {
 	switch cmd.Name() {
 	case "run":
 		if parent := cmd.Parent(); parent != nil {
-			switch parent.Name() {
-			case "k6", "cloud":
-				return true
-			}
+			return isCommandName(cmd, "cloud") || isRootCommand(parent)
```

Comment:
I think the condition is different now, as the previous code was comparing whether `cmd.Parent().Name() == "cloud"` while now it compares if `cmd.Name() == "cloud"`.

```suggestion
return isCommandName(parent, "cloud") || isRootCommand(parent)
```